panel/src/components/Layout/Stats.vue

```diff
@@ -27,14 +25,27 @@ export default {
 		}
 	},
 	methods: {
```

Comment:
I think the method naming of `isLink` (suggests a boolean) and `onClick` (suggest a handler) now doesn't really match the purpose of these methods. What about `component` and `target`?

Reply:
I like 👍